src/main/java/net/sf/jabref/logic/preferences/LastFocusedTabPreferences.java

```diff
@@ -0,0 +1,40 @@
+/*  Copyright (C) 2003-2015 JabRef contributors.
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package net.sf.jabref.logic.preferences;
+
+import java.io.File;
+
+import net.sf.jabref.JabRefPreferences;
+
+public class LastFocusedTabPreferences {
+
+    private final JabRefPreferences preferences;
+
+
+    public LastFocusedTabPreferences(JabRefPreferences preferences) {
+        this.preferences = preferences;
+    }
+
+    public void setLastFocusedTab(File file) {
+        preferences.put(JabRefPreferences.LAST_FOCUSED, file.getAbsolutePath());
+    }
+
+    public boolean hadLastFocus(File file) {
+        String lastFocusedDatabase = preferences.get(JabRefPreferences.LAST_FOCUSED);
+        return file.getAbsolutePath().equals(lastFocusedDatabase);
+    }
```

Comment:
Nice! Last comment: I think we should make this class also null-reference robust. I suggest to use Objects.requireNonNull for the preferences and for the file references just provide a special routine in case null is set. Why? At the moment, we have so many cases in which a reference can be null, I rather would avoid that this code is the cause of another bug report.